test/k8sT/manifests/testpods.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Is there something unique / different about these pods vs. some of the other sample apps we have? I would prefer to minimize the number of new different manifests we use if possible, so that if we need to debug the test, we don't have to spend as much time understanding what the manifest does compared with the deployments we use in other tests. Of course, feel free to disregard this point if there's some obvious difference in this manifest that I'm missing.

Reply:
I could not find any, but maybe I missed it? The distinguish characteristic is that these are pods (i.e., not deployments) and they use the netperf container (technically, netcat but we typically use the netperf container for that). 

The closest one I found was `test-cli.yaml` but this one has the pods running on the same node instead of different nodes which was what I wanted. (it probably does not make a difference, but since we are getting two nodes up anyway, I felt it's safer to test with packets going over the wire.)